static/bidder-params/interactiveOffers.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Interactive Offers Adapter Params",
+  "description": "A schema which validates params accepted by Interactive Offers adapter",
+  "type": "object",
+  "properties": {
+    "pubid": {
+      "type": "integer",
+      "description": "The publisher id"
+    },
+    "tmax": {
```

Comment:
There is already a TMax value on the OpenRTB request. Why does your adapter need its own?

Reply:
TMax was removed, the md file was updated